tests/phpunit/api/v3/ContributionTest.php

```diff
@@ -1794,9 +1794,12 @@ public function testCheckTaxAmount() {
     $this->callAPISuccess('contribution', 'completetransaction', array(
        'id' => $contribution['id'],
        'trxn_id' => '777788888',
+       'fee_amount' => '6.00',
     ));
-    $contribution2 = $this->callAPISuccess('contribution', 'get', array('id' => $contribution['id'], 'return' => 'tax_amount', 'sequential' => 1));
+    $contribution2 = $this->callAPISuccess('contribution', 'get', array('id' => $contribution['id'], 'return' => array('tax_amount', 'fee_amount', 'net_amount'), 'sequential' => 1));
     $this->assertEquals($contribution1['values'][0]['tax_amount'], $contribution2['values'][0]['tax_amount']);
+    $this->assertEquals('6.00', $contribution2['values'][0]['fee_amount']);
+    $this->assertEquals('94.00', $contribution2['values'][0]['net_amount']);
```

Comment:
@eileenmcnaughton this maybe wrong, I'm not sure if the net amount should essentially be tax included or without, if included then it should be 99 without tax its 94 (105.00 total v 100 total)